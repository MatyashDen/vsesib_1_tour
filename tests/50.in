10 19 9 9
3 0
3 13
4 12
4 16
7 13
7 18
8 8
9 1
9 2
