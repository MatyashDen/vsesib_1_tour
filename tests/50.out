418
0RLE0DUE0UDE0RRLLE0RDULE0RUDLE0DDUUE0UUDDE0RRRLLLE0RRDULLE0RRUDLLE0RDDUULE0RUUDDLE0DDDUUUE0UUUDDDE0RRRRLLLLE0RRRDULLLE0RRRUDLLLE0RRDDUULLE0RRUUDDLLE0RUUUDDDLE0RRRRRLLLLLE0RRRRDULLLLE0RRRRUDLLLLE0RRRDDUULLLE0RRRUUDDLLLE0RRUUUDDDLLE0RRRRRRLLLLLLE0RRRRRDULLLLLE0RRRRRUDLLLLLE0RRRRDDUULLLLE0RRRRUUDDLLLLE0RRRUUUDDDLLLE0RRRRRRUDLLLLLLE0RRRRRUUDDLLLLLE0RRRRUUUDDDLLLLE0RRRRRRUUDDLLLLLLE0RRRRRUUUDDDLLLLLE0RRRRRRUUUDDDLLLLLLE
1RLE1LRE1DUE1UDE1RRLLE1RDULE1RUDLE1LLRRE1LUDRE1DDUUE1UUDDE1RRUDLLE1RDDUULE1RUUDDLE1LLLRRRE1LLUDRRE1LUUDDRE1UUUDDDE1RRUUDDLLE1RUUUDDDLE1LLLLRRRRE1LLLUDRRRE1LLUUDDRRE1LUUUDDDRE1RRUUUDDDLLE1LLLLLRRRRRE1LLLLUDRRRRE1LLLUUDDRRRE1LLUUUDDDRRE1LLLLLLRRRRRRE1LLLLLUDRRRRRE1LLLLUUDDRRRRE1LLLUUUDDDRRRE1LLLLLLUDRRRRRRE1LLLLLUUDDRRRRRE1LLLLUUUDDDRRRRE1LLLLLLUUDDRRRRRRE1LLLLLUUUDDDRRRRRE1LLLLLLUUUDDDRRRRRREWWWWWWWWWWWWWWWWWWWWWWWW
2LRE2DUE2LLRRE2LDURE2DDUUE2LLLRRRE2LLDURRE2LDDUURE2LLLLRRRRE2LLLDURRRE2LLDDUURRE2LLLLLRRRRRE2LLLLLLRRRRRREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
3RLE3LRE3DUE3UDE3RRLLE3RDULE3RUDLE3LDURE3DDUUE3UUDDE3RRUDLLE3RUUDDLE3LDDUURE3UUUDDDE3RRUUDDLLE3RUUUDDDLE3UUUUDDDDE3RRUUUDDDLLE3RUUUUDDDDLE3RRUUUUDDDDLLEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
4RLE4LRE4DUE4UDE4RRLLE4RDULE4RUDLE4LLRRE4LDURE4DDUUE4RRDULLE4RDDUULE4LLLRRRE4LLDURRE4LDDUURE4RRDDUULLE4LLDDUURREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
5LRE5DUE5UDE5LLRRE5LDURE5LUDRE5DDUUE5UUDDE5LLDURRE5LDDUURE5LLDDUURREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
6RLE6LRE6DUE6UDE6RRLLE6RDULE6RUDLE6LLRRE6LDURE6LUDRE6UUDDE6RRDULLE6RUUDDLE6LLLRRRE6LLDURRE6LLUDRRE6LUUDDRE6UUUDDDE6LLLUDRRRE6LLUUDDRRE6LUUUDDDRE6LLLUUDDRRRE6LLUUUDDDRRE6LLLUUUDDDRRREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
7LRE7UDE7LUDRE7UUDDE7LUUDDRE7UUUDDDEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
8RLE8UDE8RRLLE8RUDLE8UUDDE8RRRLLLE8RRUDLLE8RUUDDLE8UUUDDDE8RRUUDDLLE8RUUUDDDLE8RRUUUDDDLLEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
